src/collective/fingerpointing/logger.py

```diff
@@ -8,48 +8,71 @@
 import zc.lockfile
 
 
-logger = logging.getLogger(PROJECTNAME)
```

Comment:
this logger must not be removed as is used for logging to the event log and it's different from the audit log.

Reply:
Thats fine and by intend.

The ``logging.getLogger(PROJECTNAME) always return the same logger (singleton for each name, see python docs).
So thats why I introduced commonlogger for event-log only logging (with a different name).

The logger in the LogInfo class has the event-log logger still attached, so its always logs to both locations.